doc/source/metadata/Probabilities.rst

```diff
@@ -0,0 +1,346 @@
+.. _prob-section:
+
+Probabilistic Information
+=========================
+
+.. contents:: Contents
+    :depth: 3
+
+Context
+-------
+
+As IMPROVER is inherently probabalistic,
+it seems appropriate to have a section specifically focussed
+on the represntation of probabalistic information in the metadata.
+This section also looks at the use of the `CF Metadata Conventions`_ 
+cell methods which are also heavily used within IMPROVER.
+
+
+Representation of probabilistic information
+-------------------------------------------
+
+Background
+**********
+
+Probabilistic information can be represented in three different ways: 
+
+* Ensemble members (realizations or scenarios) 
+* Probabilities of being above, below or between a set of thresholds 
+* Percentile values, representing the value of the diagnostics 
```

Comment:
Whilst I agree my sentence is not really a definition, just a statement of what the data looks like from a technical perspective, I'm not quite comfortable with your definition (partly because is contains the word "observed", which can be interpreted in ab unintended way), so I've suggested "...representing thresholds of the distribution at or below which a given percentage of values fall."